deployed.gomplate

```diff
@@ -4,11 +4,9 @@
 {{- if and (le $linecount 1) (eq $inputlen 0) -}}
 No commits
 {{- else -}}
-{{ print ":" (datasource "emojis").head "-" ((datasource "emojis").colours | random.Item) ":" "   " }}   
```

Comment:
This is me being lazy I can implement the changes here too but I figured it's not going to Slack so doesn't need to be as fancy